docs/features/nixl_connector_usage.md

```diff
@@ -0,0 +1,154 @@
+# NixlConnector Usage Guide
+
+NixlConnector is a high-performance KV cache transfer connector for vLLM's disaggregated prefilling feature. It provides fully asynchronous send/receive operations using the NIXL library for efficient cross-process KV cache transfer.
+
+## Prerequisites
+
+### Installation
+
+Install the NIXL library:
+
+- Refer to [NIXL official repository](https://github.com/ai-dynamo/nixl) for installation instructions
```

Comment:
we just switched to `uv pip install nixl` in our build (check Dockerfile), I think we can safely suggest that and refer to the page for building from source instructions.